api/pkg/dbtest/containertest.go

```diff
@@ -0,0 +1,53 @@
+package dbtest
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func StartMongoDBContainer(t *testing.T) (string, func()) {
+	req := testcontainers.ContainerRequest{
+		Image:        "mongo:latest",
+		ExposedPorts: []string{"127.0.0.1:0/tcp"},
+		WaitingFor:   wait.ForListeningPort("127.0.0.1:0/tcp"),
+	}
```

Comment:
Testcontainer makes available some modules, that are, essentially, containers ready to use with some technologies. MongoDB is one of them.

I think you could use it.

https://testcontainers.com/modules/mongodb/

Reply:
I will mark it as resolved due to the new changes made.